sdk/cosmos/azure-cosmos/CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 ## Release History
 
+### 4.46.0-beta.1 (Unreleased)
```

Comment:
will also need to update the azure-cosmos sdk version in pom.xml to 4.46.0-beta.1, else the pipeline will fail

and then you will need to also update all the versions mentioned in 
`{x-version-update;com.azure:azure-cosmos;dependency} `
`{x-version-update;com.azure:azure-cosmos;current}`

Reply:
I can follow up on this in the release branch / PR.

Reply:
Ran the prepare release script to update required files to specify 4.45.3-hotfix.